### DISCUSSION TITLE
Categorized Options

### DISCUSSION THREAD

**Discussion by theresa-de-ocampo** [koolamusic/chakra-ui-autocomplete#99] [Ideas]
Provision for categorized options would be great. For example, if I have an autocomplete field for skills, I would want to present the options like so:

* Industry Knowledge
    * Object-Oriented Programming
    * Front-End Development
    * Back-end Development
    * Project Management
* Tools and Technologies
    * ReactJS
    * Laravel
    * MySQL

Presentation similar to that of Chakra UI's `MenuGroup`.

**Comment by koolamusic** [comment]
@theresa-de-ocampo check out the `customItemRenderer` prop here is an example https://github.com/koolamusic/chakra-ui-autocomplete#usage-example-with-custom-item-renderer

**Reply by bloom-fire** [reply]
I thought of using `customItemRenderer` for categorized options as well. But I can't see how it will work out, since we need to render items **by group**. I also don't want categories to be an actual item that can be selected, since it's purely just a divider.

**Reply by koolamusic** [reply]
Kindly share an example data let me try make a code sandbox for you
